src/k-provider/ovp/play-source-url-builder.js

```diff
@@ -26,7 +26,7 @@ export default class PlaySourceUrlBuilder {
     const {partnerId, entryId, ks, uiConfId, format, protocol, extension, flavorIds} = urlParams;
 
     //verify mandatory params
-    if (!serviceUrlOrigin || !partnerId || !entryId || !format || !protocol) {
+    if (!serviceUrlOrigin || !(partnerId || partnerId === 0) || !entryId || !format || !protocol) {
```

Comment:
```suggestion
    if (!serviceUrlOrigin || !(partnerId > -1) || !entryId || !format || !protocol) {
```

would be shorter

Reply:
I don't want to break the partnerId as string option

Reply:
it can be both numeric and string ?

Reply:
per the docs it should be a number but string also works so i dont want to break it if an app use it